packages/ui/src/utils/component-options/types.ts

```diff
@@ -1,4 +1,15 @@
-import type { ComponentOptionsBase, PropType, ExtractPropTypes } from 'vue'
```

Comment:
We don't have props in docs for now.

<img width="981" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/f3c00208-883e-4c6d-aa07-36893d328c03">